modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/plugins/fields-sidebar/components/FieldsSidebar.es.js

```diff
@@ -119,8 +119,8 @@ export const DataEngineFieldsSidebar = ({title}) => {
 					'fieldAdded',
 					dropLayoutBuilderField({
 						addedToPlaceholder: true,
-						dataLayoutBuilder,
 						fieldTypeName: name,
+						fieldTypes: dataLayoutBuilder.props.fieldTypes,
```

Comment:
I think We can get those fieldTypes from useConfig :)

Reply:
This is the old `FieldsSidebar.es.js`, we should not change it